package.json

```diff
@@ -16,76 +7,57 @@
     "type": "git",
     "url": "https://github.com/skatejs/skatejs"
   },
-  "main": "dist/index.js",
-  "jsnext:main": "src/index.js",
+  "main": "lib/index.js",
```

Comment:
hmm well and that's the thing, we don't need specific commonjs build, I think main should point to UMD build